students/gyuchan/users/views.py

```diff
@@ -12,29 +13,31 @@ def post(self, request):
         try:
             check_email    = "^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$"
             check_password = "^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$"
-            
+
             user_name         = data['name']
             user_email        = data['email']
             user_password     = data['password']
             user_phone_number = data['phone_number']
-            
+
             if User.objects.filter(email=user_email).exists():
-                return JsonResponse({'message':'ALREADY_EXISTS'}, status=400)
+                return JsonResponse({'message': 'ALREADY_EXISTS'}, status=400)
             if not re.match(check_email, user_email):
-                return JsonResponse({'message':'EMAIL_NOT_MATCH'}, status=400)
+                return JsonResponse({'message': 'EMAIL_NOT_MATCH'}, status=400)
             if not re.match(check_password, user_password):
-                return JsonResponse({'message':'PASSWORD_NOT_MATCH'}, status=400)
+                return JsonResponse({'message': 'PASSWORD_NOT_MATCH'}, status=400)
+
+            hashed_password = bcrypt.hashpw(user_password.encode('utf-8'), bcrypt.gensalt())
+            decoded_password = hashed_password.decode('utf-8')
```

Comment:
```suggestion
            hashed_password = bcrypt.hashpw(user_password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```
활용되지 않을 변수를 사용하는 것이 큰 의미가 있지는 않은 것 같아요.
hashpw() 메소드 뒤에 디코드만 해주는 코드라서 그렇게 가독성이 좋지 않은 줄도 아닌 것 같구요.
만약 정말 가독성을 생각했다면 다른 방식으로 접근해볼 수 있을 것 같습니다.
```suggestion
            hashed_password = bcrypt.hashpw(
                user_password.encode('utf-8'), 
                bcrypt.gensalt()
            ).decode('utf-8')
```